client/transaction-pool/src/lib.rs

```diff
@@ -28,7 +28,7 @@ mod metrics;
 
 pub mod error;
 
-#[cfg(any(feature = "test-helpers", test))]
+#[cfg(test)]
```

Comment:
Changing this means that we can't use the stuff in `testing` outside of this crate. Not sure if it is used though.

Reply:
[See this comment](https://github.com/paritytech/substrate/pull/6267#issuecomment-639541465) :D